Bigtable/src/Instance.php

```
@@ -0,0 +1,313 @@
+     *
+     * When instances are created or updated, they may take some time before
+     * they are ready for use. This method allows for checking whether an
+     * instance is ready.
+     *
+     *
+     * @param array $options [optional] Configuration options.
+     */
+    public function state(array $options = [])
+    {
+        throw new \BadMethodCallException('This method is not implemented yet');
+    }
+
+    /**
+     * Create a new instance.
+     *
+     * Example:
+     * ```
+     * $bigtable = new Google\Cloud\Bigtable\BigtableClient();
+     * $operation = $instance->create(
+     *     $bigtable->clusterMetadata(['clusterId' => 'my-cluster', 'locationId' => 'us-east1-c'])
+     * );
+     * ```
+     *
+     * @codingStandardsIgnoreStart
+     * @see https://cloud.google.com/bigtable/docs/reference/admin/rpc/google.bigtable.admin.v2#CreateInstanceRequest CreateInstanceRequest
+     * @codingStandardsIgnoreEnd
+     *
+     * @param array $clusterMetadataList [
+     *        $bigtable->clusterMetadata('clusterId1', 'locationId1', 'SSD', 3),
+     *        $bigtable->clusterMetadata('clusterId2', 'locationId2', 'SSD', 3)
+     * ]
+     * @param string $displayName **Defaults to** the value of $instanceId.
+     * @param array $labels as key/value pair ['foo' => 'bar']. For more information, see
+     *        [Using labels to organize Google Cloud Platform resources](https://cloudplatform.googleblog.com/2015/10/using-labels-to-organize-Google-Cloud-Platform-resources.html).
+     * @param int $type Possible values are represented by the following constants:
+     *        `Google\Cloud\Bigtable\Instance::INSTANCE_TYPE_PRODUCTION`,
+     *        `Google\Cloud\Bigtable\Instance::INSTANCE_TYPE_DEVELOPMENT` and
+     *        `Google\Cloud\Bigtable\Instance::INSTANCE_TYPE_UNSPECIFIED`.
+     *        **Defaults to** using `Google\Cloud\Bigtable\Instance::INSTANCE_TYPE_UNSPECIFIED`.
+     *
+     * @return LongRunningOperation<Instance>
+     */
```

Should we add `@throw` here for InvalidArgumentException?